Makefile

```diff
@@ -199,3 +202,14 @@ $(LICHE):
 
 $(PROMU):
 	$(call fetch_go_bin_version,github.com/prometheus/promu,$(PROMU_VERSION))
+
+$(PROTOC):
+	@echo ">> fetching protoc@${PROTOC_VERSION}"
+	@if [ ! -x '$(TMP_GOPATH)/bin/protoc' ]; then \
+		cd -- $(TMP_GOPATH) && curl -OLSs $(PROTOC_DOWNLOAD_URL); \
+		unzip -qq $(PROTOC_PACKAGE); \
+	fi
+	@echo ">> installing protoc@${PROTOC_VERSION}"
+	@mv -- "$(TMP_GOPATH)/bin/protoc" "$(BIN_DIR)/protoc-$(PROTOC_VERSION)"
+	@echo ">> produced $(BIN_DIR)/protoc-$(PROTOC_VERSION)"
+	@go install ./vendor/github.com/gogo/protobuf/protoc-gen-gogofast
```

Comment:
Let's run thins in very beginning. Otherwise when you run `protoc` and it will get into to the 214 line and we abort this, protoc will be installed so running `make proto` will not install protoc, but `protoc-gen-gogofast` won't be there.

Reply:
You are right, my bad.